Api/Endpoints/DancerEndpoints/GetById.cs

```diff
@@ -25,9 +25,15 @@ public GetById(IDancerService dancerService)
             OperationId = "Dancers.GetById",
             Tags = new[] { "Dancers" })
         ]
-        public override async Task<ActionResult<GetDancerByIdResponse>> HandleAsync([FromRoute] GetDancerByIdRequest request, CancellationToken cancellationToken = new CancellationToken())
+        public override async Task<ActionResult<GetDancerByIdResponse>> HandleAsync([FromRoute] GetDancerByIdRequest request, CancellationToken cancellationToken = new())
         {
-            var dancer = await _dancerService.GetDancerByIdAsync(request.Id, cancellationToken);
+            var dancerResult = await _dancerService.GetDancerByIdAsync(request.Id, cancellationToken);
+            if (!dancerResult.IsSuccess)
```

Comment:
Is there a check for what Status the result is returning?

Reply:
Yep, `result.Status`